src/main/java/de/adesso/anki/battle/world/bodies/Mine.java

```diff
@@ -2,17 +2,32 @@
 
 import com.states.GameState;
 import de.adesso.anki.battle.mqtt.MqttService;
+import de.adesso.anki.battle.util.Position;
+import de.adesso.anki.battle.world.Body;
 import de.adesso.anki.battle.world.DynamicBody;
+import de.adesso.anki.battle.world.World;
+
 import org.eclipse.paho.client.mqttv3.MqttException;
 
 import java.util.List;
 
 public class Mine extends DynamicBody {
-
+	private long timer ; 
+	private boolean active;

```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "active" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1068)